api/src/main/scala/play/twirl/api/Formats.scala

```diff
@@ -93,9 +58,7 @@ object HtmlFormat extends Format[Html] {
   /**
    * Creates a safe (escaped) HTML fragment.
    */
-  def escape(text: String): Html = {
-    new Html(Nil, text, true)
-  }
+  def escape(text: String): Html = Html(HtmlEscape.escapeHtml4(text))
```

Comment:
Note: this *does* escape apostrophes.

The difference of HTML4 vs HTML5 is what named character references will be used (HTML5 having more).